CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Changed
 
 - Manage xapi request without consumer_site
+- Update live sessions API to use nested video ID route
```

Comment:
Should be moved in the `Unreleased` section

Reply:
It is already, isn't it?